### DISCUSSION TITLE
ContentDialog position

### DISCUSSION THREAD

**Discussion by RomanCherkasov** [bdlukaa/fluent_ui#1132] [Q&A] [answered]
Hello there!
I added a TextBox to the ContentDialog. And this created the problem that when running on mobile devices. The keyboard overlaps the control buttons.

Now I have solved this problem by changing the source code of the ContentDialog class. (Changed alignment: Alignment Directional.center to alignment: Alignment Directional.topCenter)

Is there any more correct way to achieve a similar effect, or any other workaround?

**Comment by bdlukaa** [comment, marked as answer]
I believe it is better to add the view inset margin to the dialog. Microsoft only describes the dialog position on the center.

**Reply by RomanCherkasov** [reply]
Ok thx!
